### ISSUE TITLE
julia-debug aborts when using Images package

### ISSUE THREAD

**Issue by JohDo321** [JuliaLang/julia#18305]
When trying to use the Julia package Images with julia-debug, the process crashes with the following message:
```
julia> using Images
julia-debug: dump.c:2117: jl_recache_type: Zusicherung »t->uid != 0« nicht erfüllt.

signal (6): Abgebrochen
gsignal at /lib/x86_64-linux-gnu/libc.so.6 (unknown line)
abort at /lib/x86_64-linux-gnu/libc.so.6 (unknown line)
...
```
I've tried to use a self compiled Julia version as well as the original binaries from the home page: 
Version 0.4.6 (2016-06-19 17:16 UTC) 

The system runs with a x64 Ubuntu 16.04.1 OS

**Comment by vtjnash** [comment]
duplicate of https://github.com/JuliaLang/julia/issues/16301 & related issues (https://github.com/JuliaLang/julia/issues/12096, https://github.com/JuliaLang/julia/issues/17809, https://github.com/JuliaLang/julia/pull/16488). not sure what the intention for backporting those to v0.4 was?

**Comment by tkelman** [comment]
Adding backport pending to an issue isn't helpful if there's a corresponding PR. Which should it be?

**Comment by tkelman** [comment]
Again, please be as specific as possible. Don't put backport pending on this issue, which does not have any commits in or linking to it. Put backport pending on one or multiple of the PR's (or if it's a commit that went straight to master, at least link to it) if you'd like me to evaluate them.
